DictionaryTags.txt

```diff
@@ -690,7 +668,16 @@ vol_block_size          4014	int32		card	"Block size in bytes"
 vol_directory           4015	void		-	"Contents of the volume in a special format; the data type will be FIFF_VOID"
 #<free>                 4016
 #...			...
-#<free>                 5000
+#<free>                 4099
+
+#==============================================================================================================
+:channel info extension
+#==============================================================================================================
+#<free>			4100
+ch_coord_frame		4101	int32      -	"Channel coordinate frame"
```

Comment:
This is one thing that *is* missing from the channel info. Currently the `coord_frame` is taken based on the channel type (MEG: device, EEG: head), but it's cleaner just to encode it directly. We're out of constants in the 250-ish range, so I reserved 100 slots for volume (above) and started a new potential range for channel information or other info in the next free block.

Reply:
Very good point. An explicit coordinate frame is definitely better than implicit. I have nothing against that. New things should use it. Unfortunately if somebody needs backwards compatibility, the reader needs to know the implicit ones.
There seem to be some space available around 350+ (I hope we never have tens of different ways to encode data).
This is also the correct time to specify exactly what each thing means, and think about is this compatible with future. This probably goes to the other pull, but there should be convenient way to have more text on the descriptions.

Reply:
> Unfortunately if somebody needs backwards compatibility, the reader needs to know the implicit ones.

I think this part is okay -- existing readers already implement this logic. So it should be backward compatible, and any new readers will need to be compatible with the (many, many) files without a `FIFFB_CH_INFO` block anyway.

> There seem to be some space available around 350+ 

I'll push a commit to move them down there.

> This is also the correct time to specify exactly what each thing means, and think about is this compatible with future.

At least these two additions seem fairly clear/unambiguous in the context of channel information. But I'll make a comment in the other issue about this more general problem.